### DISCUSSION TITLE
和不適合的朋友說再見

### DISCUSSION THREAD

**Discussion by Hsucuchia** [1101-GS4719-C/Python-Community-GS4719-C#1214] [General]
長大的過程中，不是每個人都能夠一直陪伴我們，有時也要學習和不適合的朋友說再見。
如果在一段關係中，總是只有自己在付出，或是對方總是令你感到失望；也可能因為時空間的變化，讓原本很好的彼此出現隔閡。
或許是時候該問問自己，是不是應該放下這些曾經的好友們，將時間留給更重要的人事物，或許會感到愧疚，當一段友情不能讓自己變得更好時，試著放下也沒有錯，只是這段友情沒那麼適合自己了。
朋友重質不重量，朋友並不是多就好，合不合得來更為重要。適時地篩選朋友，想想和誰相處才是舒服、快樂的，會在自己難過時願意陪伴自己，那個人會更適合成為長久的朋友!

**Comment by GuanYann** [comment]
成長中，有些友情只是短暫的相遇。當一段友情讓自己變得不快樂時，應學會放下，找到更合適的友誼。讓時間留給更重要的人和事，才能讓人生更充實和精彩。

**Comment by zilinng** [comment]
我覺得朋友真的重精不在多，當和一個人接觸時負面心情大於正面的時候我會選擇慢慢疏遠，對我來說學會篩選自己身邊的人才能擁有更好的生活體驗。

**Comment by Chienyuuu6** [comment]
我們常常覺得堅持到底才是對的態度，然後懂得「放下、放棄」是比堅持下去還要困難的事！很多時候我們真的要灑脫的「放下某些執念」才能讓自己的心態、生活達到更好的境界。我自己過去給自己設定一些目標，搞到後來會因為沒有達成目標而覺得焦慮不安，那些焦慮感甚至大大影響到我的生活和健康，但事實上沒跟著目標走根本不會世界末日🤣

隨著後來練習「放下」，我的狀況也越來越好～這樣的概念可以套用在很多層面上，人際也是！所以我們真的不要跟不適合自己的人事物勾勾迪

**Comment by CCello** [comment]
你所說的很對。和不適合的朋友保持關係往往會讓我們感到疲憊和不滿，甚至會對我們的情緒和生活產生負面影響。和真正合適的朋友在一起，不僅可以帶來愉悅，還可以互相幫助和支持。

然而，要放下一段友誼也不是一件容易的事情。我們會擔心失去這段友誼會讓自己變得更孤單、更無助。但是，如果我們一直忍耐著不合適的朋友，最終可能會帶來更大的痛苦和後悔。因此，當我們意識到一段友誼不再合適時，我們應該勇敢地面對，適時地放下，並尋找更合適的朋友。

**Comment by tower20021202** [comment]
和不合適的人相處真的要能盡快斷乾淨，有時其實不是對方不好或是你不好，可能就只是彼此只之間不太合適，與他相處會消耗自己的許多精力讓你變得無所適從。
當然要隨便放下朋友時常會覺得很難受，但只要放長期來看，這些你明知會對你有害的朋友就算現在離開他對我們自身並不會造成任何的損失，因此我們須勇敢地放下，未來便能真心的感謝自己

**Comment by Jack-1D** [comment]
確實，有時候不該強求自己接受所有事情，應該還是要以自己過得舒適，又不會影響別人為主。我認為偶爾和自己相處也是一件很重要的事。把自己當成最知心的朋友可以減少與他人的摩擦，等自己準備好了，再用最真誠的一面面對別人。

**Comment by lyien1021** [comment]
以前的我都以為能跟每個人都能成為朋友，不過後來我漸漸發現，真的不用需要浪費力氣討好每個人，個性、價值觀不合還硬要待在一起的話，其實就是一直在互相消耗彼此的精神罷了。

**Comment by YoYooHuang** [comment]
其實哪些朋友對自己好，哪些人適合當自己的朋友我們都心知肚明。但也沒必要鬧翻，這些都自己知道就好了。我覺得，如果去在意那些你討厭的人，那對那些喜歡你的人是非常不公平的。與其討厭別人，不如好好珍惜自己擁有的一切。

**Comment by pigeon9975** [comment]
沒錯，如果跟那位「朋友」相處起來覺得很有壓力，何不放過自己，漸漸遠離他讓自己過得輕鬆點呢？
我覺得漸漸疏離並不代表吵架，也不代表關係變得不好，只是相處模式有所變化而已，我想，讓彼此都保有自己舒服的空間才是友情最重要、也最珍貴的地方！

**Comment by ziihui** [comment]
非常認同你的看法！友情的重要性不在於數量，而在於質量。與那些能夠真正陪伴、支持自己的人交往，比和許多表面上看起來很熟的人維持虛假的關係更加重要。有時候，放下一些曾經的友情並不是出於無情或冷漠，而是為了更好地照顧自己和自己身邊的人。與那些真正適合自己的人一起度過時光，將會帶給我們更多的快樂和成長。

**Comment by llh26** [comment]
我常常覺得自己要努力維持好每一段關係，但到頭來發現我快把自己累死，我會太在意我說出來的話會不會在無意中傷害到別人，或者太在意別人會怎麼想我，但卻從未好好去思考對方是否值得我這麼做，我覺得審視一段關係真的非常重要，也不是每個人都能做出正確的判斷，希望大家都不要忽視自己的感受，能把自己放優先，把不良的交友關係都盡早處理好💪🏻

**Comment by JessyCheng2003** [comment]
如果我遇到個性不合或相處不自在的朋友，我會慢慢遠離，但還是會維持基本的關係，不要樹敵
。

**Comment by Miinteaa** [comment]
在上了大學之後，我發現自己對於人際關係的執著好像沒有以前深了，小時候都希望自己受到每個人的青睞，但這樣會過度在意別人的看法而不能好好做自己，也可能有部分原因是因為在大學生活中大家都有各自要做的事情，沒有那麼多時間專注於對於自己來說不太重要的人事物，總之，也算是我某部分的成長～

**Comment by pudding0803** [comment]
天下沒有不散的筵席。我最近開始認為，人生就是要不斷地學習接受並釋懷周遭所發生的一切。另外，畢竟是自己的人生，自己的感受、自己的想法應該會是比較優先的順位，正如你所說的，如果相處起來會感到不適，或是有其他負面情緒，那或許還是不要硬撐比較好。

**Comment by Alexgns** [comment]
有時與不適合的朋友說再見是為了保護自己的健康和幸福。我們應該與那些能夠真正支持我們成長的人建立深厚的友誼。雖然這可能是一個艱難的決定，但放下不適合的關係能夠為我們打開全新的機會和更有意義的人際關係。

**Comment by zhiron544** [comment]
我是覺得與其一開始就去免強自己與不合適的人、事、物相處，不如斷的乾淨，一次解脫

**Comment by Aaron0810** [comment]
慎選朋友確實重要，但說實在很多朋友是熟了之後才會展現真實的自己，假設在那時候你才發現你們不合的話其實也很難斷乾淨，即便你想跟他說再見也很難，除非你狠的下心完全切割，不過我覺得正常人應該很難做到這個。

**Comment by xiang305** [comment]
慎選朋友確實很重要，但那些日常生活中就會見到的人，例如室友之類的，也沒辦法說放下就放下。如果你特意疏遠他的話，整個寢室的氣氛就變得很尷尬。我認為要如何解決這個狀況仍是一個難題
